SwiftVideoBackground/Sources/VideoBackground.swift

```diff
@@ -176,8 +188,24 @@ public class VideoBackground {
         playerLayer.player?.play()
     }
 
+    /// Generate an image from the video to show as thumbnail
+    public func getThumbnailImage(from url: URL, at time: CMTime) -> UIImage? {
+        let asset = AVAsset(url: url)
+        let imageGenerator = AVAssetImageGenerator(asset: asset)
+
+        do {
+            let thumbnailImage = try imageGenerator.copyCGImage(at: time, actualTime: nil)
+            return UIImage(cgImage: thumbnailImage)
+        } catch {
+            print(error)
```

Comment:
Could you make this throw instead of just print? This will allow our users to gracefully handle failures.